packages/VendorLocker/NodeVendorLocker/ClassMethodReturnTypeOverrideGuard.php

```diff
@@ -102,11 +102,6 @@ private function isReturnTypeChangeAllowed(ClassMethod $classMethod): bool
             return true;
         }
 
-        $scope = $classMethod->getAttribute(AttributeKey::SCOPE);
```

Comment:
the actual scope drop. everything else was turning `AbstractRector` into `AbstractScopeAwareRector`